### PR TITLE
Fix loadChain()

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Fix loadChain by adding getNumBlockFiles()
- Use short names: 'hashToNumber' -> 'h2n'
- Remove some dead code

Related issue: https://github.com/ainblockchain/ain-blockchain/issues/509

### CODE REVIEW

File: common/file-util.js [hunk 1]
```diff
@@ -143,34 +146,50 @@ class FileUtil {
...
-      fs.writeFileSync(hashToNumberPath, blockNumber);
+      fs.writeFileSync(h2nPath, blockNumber);
     } else {
-      logger.debug(`${hashToNumberPath} file already exists!`);
+      logger.debug(`${h2nPath} file already exists!`);
+    }
+  }
+
+  static deleteH2nFile(chainPath, blockHash) {
+    logger.info(`Deleting h2n file with block number: ${blockHash}`);
```
Comment (liayoo) [inline common/file-util.js:175]: number -> hash

**Review by liayoo — APPROVED** [review]
LGTM with a minor comment. Thanks!

File: common/file-util.js (hunk 1 above)
Comment (platfowner) [reply common/file-util.js]: Fixed, thanks!

**Comment by platfowner** [comment]
Thanks for the prompt review, now merging..

**Pull request merged**